src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/TypeDescriptor.cs

```diff
@@ -26,8 +28,8 @@ public sealed class TypeDescriptor
         // every call to this class, so it will be created soon after
         // class load anyway.
         private static readonly WeakHashtable s_providerTable = new WeakHashtable();     // mapping of type or object hash to a provider list
-        private static readonly Hashtable s_providerTypeTable = new Hashtable();         // A direct mapping from type to provider.
```

Comment:
I'm not sure I can trivially prove the changes in this file are correct. Can we revert this file's changes?